utils/jsonapi.js

```diff
@@ -78,8 +77,7 @@ const serializerOptions = (serializerArgs) => {
       currentPageSize: pageSize,
     };
   }
-
-  return options;
+  return { ...options, ...relationships }; // concat relationships to options
```

Comment:
Could you format this comment based on the airbnb style guide? https://github.com/airbnb/javascript#comments--singleline